.github/workflows/docs.yml

```diff
@@ -0,0 +1,47 @@
+name: Kymatio docs CI
+
+on:
+    - push
+    - pull_request
+
+jobs:
+    build:
+        runs-on: ubuntu-20.04
+
+        steps:
+            - uses: actions/checkout@v1
+            - name: Set up Python
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.7
+            - name: Set up TeX Live
+              run: |
+                  sudo apt update
+                  sudo apt install texlive texlive-latex-extra cm-super-minimal dvipng
+            - name: Install Python dependencies
+              run: |
+                  python3 -m pip install --upgrade pip
+                  python3 -m pip install -r requirements.txt
+                  python3 -m pip install -r requirements_optional.txt
+                  pip install torch==1.8.0+cpu \
+                              torchvision==0.9.0+cpu \
+                              torchaudio==0.8.0 \
+                              -f https://download.pytorch.org/whl/torch_stable.html
+            - name: Set up Kymatio
+              run: python3 setup.py develop
+            - name: Generate documentation
+              run: |
+                  pushd doc
+                  make clean
+                  #make html
+                  mkdir -p build/html
+                  echo "haha" > build/html/index.html
```

Comment:
haha :rofl:

Reply:
Well, you gotta be able to laugh a little sometimes…